### ISSUE TITLE
issue in label encorder

### ISSUE THREAD

**Issue by zariab123** [giorgosterz/Document_CLassifier---NLP#1]
Hi i am having issue with label encoder, it says the category is not defined what should i do, is there any library for it or anything

**Comment by giorgosterz** [comment]
Hello zariab123, could you post the code and a screenshot of the error so I can get a better understanding of what went wrong here?

**Comment by zariab123** [comment]
Hi
This is the error i am encountering

Warm Regards

Zariab Fatima

**Comment by giorgosterz** [comment]
Hello,

Looks like you didn't post any picture or code with your answer. Maybe give it one more try?

Best,
George

**Comment by zariab123** [comment]
Hey George I attached the photo of error in the previous email

**Comment by zariab123** [comment]
![error](https://user-images.githubusercontent.com/48620336/96286081-80a41b80-0fe8-11eb-87bc-6068d73c03ba.PNG)

**Comment by giorgosterz** [comment]
It looks like you haven't defined 'categories' earlier. Check out my notebook at **In[5]** . There I define categories as an empty list ("categories = [ ]") and then I populate it with the labels of my text data . Define your categories too according to your data and that should solve your problem!

**Comment by zariab123** [comment]
I'm extremely sorry, I missed this part. I'm working on it right now. If I
have any other questions can I ask later on?

**Comment by zariab123** [comment]
Hi after including  In[5] im having another type of error :(  if you could get it



![image](https://user-images.githubusercontent.com/48620336/96290576-82251200-0fef-11eb-8b51-4d5f6a42c3f5.png)

**Comment by giorgosterz** [comment]
Yes, the problem is that the data you put into "categories" is in 5 columns, but that should be an 1 dimensional array. What Label Encoder does is taking some categorical values in this form ["Dog", "Cat", "Dolphin"] and transforms them to [1,2,3]. So you should modify your code so "categories" is an 1d array and keep only the labels you need.
